Still failing to fetch git-annex artifact

follow up to #72, unfortunately spreading them apart in time didn't help, we still get
```
Using curl as "curl --silent"
Getting artifacts_url from https://api.github.com/repos/datalad/git-annex/actions/workflows/build-ubuntu.yaml/runs?status=success&branch=master into /tmp/tmp.pJLFDhAJv3
Getting archive download url from https://api.github.com/repos/datalad/git-annex/actions/runs/467805073/artifacts
Getting download url from https://api.github.com/repos/datalad/git-annex/actions/artifacts/34085739/zip
Error: Process completed with exit code 1.
```

needs troubleshooting (remove --silent?) and a proper fix (if possible since may be we just exhausted some limit for cross-repos requests?) or a decide on the way forward (e.g. finally do https://github.com/datalad/git-annex/issues/2 and use that distribution instead of directly from github artifacts).

--- a/scripts/ci/download-latest-artifact
+++ b/scripts/ci/download-latest-artifact
@@ -41,7 +41,7 @@ definedOrExit "${ARCHIVE_DOWNLOAD_URL}" "Unable to get archive_download_url" ${J
 
 call_curl -i "${ARCHIVE_DOWNLOAD_URL}" >| ${JOBS_DOWNLOAD}
 echo "Getting download url from ${ARCHIVE_DOWNLOAD_URL}"
-DOWNLOAD_URL=$(grep -oP 'Location: \K.+' < ${JOBS_DOWNLOAD})
+DOWNLOAD_URL=$(grep -ioP 'Location: \K.+' < ${JOBS_DOWNLOAD})
 definedOrExit "${DOWNLOAD_URL}" "Unable to get Location header with download url" ${JOBS_DOWNLOAD}
 DOWNLOAD_URL=${DOWNLOAD_URL%$'\r'}
 rm -f ${JOBS_DOWNLOAD}

